plugins/assets/assets.py

```diff
@@ -32,7 +32,10 @@
 def add_jinja2_ext(pelican):
     """Add Webassets to Jinja2 extensions in Pelican settings."""
 
-    pelican.settings['JINJA_EXTENSIONS'].append(AssetsExtension)
+    if 'JINJA_EXTENSIONS' in pelican.settings:
+        pelican.settings['JINJA_EXTENSIONS'].append(AssetsExtension)
+    else:
+        pelican.settings['JINJA_EXTENSIONS'] = [AssetsExtension]
```

Comment:
Do we need these changes?

Reply:
The previous script didn't work for me. If it worked for you previously, this change shouldn't make a difference.